src/modules/hub.bicep

```diff
@@ -230,80 +230,93 @@ module linkedService_storageAccount 'linkedservices/ms_cm_exports.bicep' = {
 }
 
 module dataset_exports 'datasets/ms_cm_exports.bicep' = {
-  name: 'dataset_exports'
+  name: 'ms_cm_exports'
   dependsOn: [
     linkedService_storageAccount
     linkedService_keyVault
   ]
   params: {
     dataFactoryName: dataFactoryName
     linkedServiceName: storageAccountName
+    datasetName: 'ms_cm_exports'
   }
 }
 
-module dataset_csv 'datasets/ingestion_csv.bicep' = {
-  name: 'dataset_csv'
+module ingestion_csv 'datasets/ingestion_csv.bicep' = {
+  name: 'ingestion_csv'
   dependsOn: [
     linkedService_storageAccount
     linkedService_keyVault
   ]
   params: {
     dataFactoryName: dataFactoryName
     linkedServiceName: storageAccountName
+    datasetName: 'ingestion_csv'
   }
 }
 
-module dataset_parquet 'datasets/ingestion_parquet.bicep' = {
-  name: 'dataset_parquet'
+module ingestion_parquet 'datasets/ingestion_parquet.bicep' = {
+  name: 'ingestion_parquet'
   dependsOn: [
     linkedService_storageAccount
     linkedService_keyVault
   ]
   params: {
     dataFactoryName: dataFactoryName
     linkedServiceName: storageAccountName
+    datasetName: 'ingestion_parquet'
   }
 }
 
-module transform_parquet 'pipelines/transform_parquet.bicep' = {
-  name: 'transform_parquet'
+module transform_parquet 'pipelines/transform_ms_cm_exports.bicep' = {
+  name: 'transform_ms_cm_exports_parquet'
```

Comment:
3 thoughts:
1. Shouldn't this be `ms_cm_exports_transform_parquet` so all related pipelines are sorted together by container name?
2. Should we make use `transformParquet` to keep the pipeline name together?
3. Would it make sense to do both CSV and parquet transforms in the same pipeline?